dist_tables/ddl.rst

```diff
@@ -61,6 +61,30 @@ Each created shard is assigned a unique shard id and all its replicas have the s
 
 You are now ready to insert data into the distributed table and run queries on it. You can also learn more about the UDF used in this section in the :ref:`user_defined_functions` of our documentation.
 
+.. _colocation_groups:
+
+Co-Location Groups
+------------------
+
+Table rows are stored in shards across a Citus cluster. When all rows needed by a query are available in a single shard then the tables named by the query are said to be *co-located for the query*. (Tables which are co-located for the majority of queries in an application are simply called co-located tables.) Queries with co-located tables excute faster because Citus doesn't need to shuffle data over the network.
+
+When the database administrator changes the number of worker nodes in a cluster, they run the Citus shard rebalancer to redistribute shards evenly. It's important to preserve table co-location during this process to
+keep queries fast. Because tables are co-located with respect to particular queries there is no one-size-fits-all algorithm to preserve co-location, and Citus' shard rebalancer uses a cautious heuristic. It keeps rows together when there is even a possibility that their distribution columns might be joined. Specifically it keeps them together when their distribution columns have the same data type and their distribution method is the same.
+
+Shard rebalancing must take table locks for any tables it decides must stay together. The default heuristic can result in false-positives, causing unnecessary locking and slower rebalancing. Citus allows the database administrator to customize which tables should maintain co-location -- and importantly which tables should not.
+
+To control co-location groups manually use the optional :code:`colocate_with` parameter of :code:`create_distributed_table`. Left unspecified it defaults to the value :code:`default` which lumps all tables having the same distribution column type into the same co-location group.
```

Comment:
> which lumps all tables having the same distribution column type into the same co-location group

And also the same shard-count I think. Could you confirm what was documented in the related docs issue by the developer?